apps/ml/site/analysis/analysis.jag

```diff
@@ -503,8 +583,48 @@ include("../../includes/tenantAware.jag");
         }      
     }
 
+    function loadPMMLAvailability(algorithmName){
+        var status=false;
+        if(algorithmName){
+        
+            $.ajax({
+                type: "GET",
+                url: serverUrl+'/api/configs/algorithms/'+algorithmName+'/exportable?format=pmml',
+                async: false,
+                success : function(res){
+                    status = true;
```

Comment:
We are sending a 404 if pmml not supported for the algorithm. So only when pmml is available the function will enter to the success function.